<?xml  version="1.0"  encoding="UTF-8"?>
<ui  version="4.0">
  <class>Form</class>
  <widget  class="QWidget"  name="Form">
    <property  name="geometry">
      <rect>
        <x>0</x>
        <y>0</y>
        <width>800</width>
        <height>500</height>
      </rect>
    </property>
    <property  name="windowTitle">
      <string>Form</string>
    </property>

    <layout  class="QVBoxLayout"  name="vlMain">
      <item>
        
        <widget class="QWidget" name="verticalLayoutWidget">
    
            <layout  class="QHBoxLayout"  name="horizontalLayout">
            <item>
                <widget  class="QSplitter"  name="splitter">
                <property  name="orientation">
                    <enum>Qt::Horizontal</enum>
                </property>
                
                <widget  class="QWidget"  name="wViewer">
                    <property  name="sizePolicy">
                    <sizepolicy  hsizetype="Preferred"  vsizetype="Preferred">
                        <horstretch>8</horstretch>
                        <verstretch>0</verstretch>
                    </sizepolicy>
                    </property>
                    <layout  class="QVBoxLayout"  name="vlViewer">
                    <item>
                        <widget  class="QMdiArea"  name="mdiArea"/>
                    </item>
                    </layout>
                </widget>
                
                <widget  class="QWidget"  name="wOptions">
                    <property  name="sizePolicy">
                    <sizepolicy  hsizetype="Minimum"  vsizetype="Minimum">
                        <horstretch>1</horstretch>
                        <verstretch>0</verstretch>
                    </sizepolicy>
                    </property>

                    <layout  class="QVBoxLayout"  name="vlOptions">

                        <item>
                            <widget  class="QFrame"  name="wActiveDset">
                            <property  name="frameShape">
                                <enum>QFrame::StyledPanel</enum>
                            </property>
                            <property  name="frameShadow">
                                <enum>QFrame::Raised</enum>
                            </property>
                            <property  name="lineWidth">
                                <number>8</number>
                            </property>
                            
                            
                            <layout  class="QVBoxLayout"  name="vlActiveDset">
                                
                                <item>
                                <widget  class="QLabel"  name="label_activeDset">
                                    <property  name="text">
                                    <string>Active Dataset</string>
                                    </property>
                                </widget>
                                </item>

                                <item>
                                <layout  class="QHBoxLayout"  name="horizontalLayout_5">
                                    <item>
                                    <spacer  name="horizontalSpacer_5">
                                        <property  name="orientation">
                                        <enum>Qt::Horizontal</enum>
                                        </property>
                                        
                                        <property name="sizeType">
                                            <enum>QSizePolicy::Fixed</enum>
                                        </property>
                                        
                                        <property  name="sizeHint"  stdset="0">
                                        <size>
                                            <width>30</width>
                                            <height>20</height>
                                        </size>
                                        </property>
                                    </spacer>
                                    </item>
                                    <item>
                                        <widget  class="QLineEdit"  name="edit_activeDset">
                                            <property  name="text">
                                            <string> </string>
                                            </property>
                                        </widget>
                                    </item>
                                </layout>
                                </item>
                                
                            </layout>
                            </widget>
                        </item>
                        
                    <item>
                        <widget  class="QFrame"  name="wSelectCol">
                        <property  name="frameShape">
                            <enum>QFrame::StyledPanel</enum>
                        </property>
                        <property  name="frameShadow">
                            <enum>QFrame::Raised</enum>
                        </property>
                        <property  name="lineWidth">
                            <number>8</number>
                        </property>
                        
                        <layout  class="QVBoxLayout"  name="vlSelectCol">

                            <item>
                            <widget  class="QLabel"  name="label_select">
                                <property  name="text">
                                <string>Select Columns</string>
                                </property>
                            </widget>
                            </item>
                            
                            <item>
                            <layout  class="QHBoxLayout"  name="hlSelCol1">
                                <item>
                                <spacer  name="horizontalSpacer_5">
                                    <property  name="orientation">
                                    <enum>Qt::Horizontal</enum>
                                    </property>

                                    <property name="sizeType">
                                        <enum>QSizePolicy::Fixed</enum>
                                    </property>
                                    
                                    <property  name="sizeHint"  stdset="0">
                                    <size>
                                        <width>30</width>
                                        <height>20</height>
                                    </size>
                                    </property>
                                </spacer>
                                </item>
                                <item>
                                    <layout  class="QVBoxLayout"  name="vlComboSel"/>
                                </item>
                            </layout>
                            </item>
                            
                            <item>
                            <widget  class="QWidget"  name="wSelOptions">
                                <layout  class="QHBoxLayout"  name="hlSelOptions">
                                    <item>
                                    <spacer  name="horizontalSpacer">
                                        <property  name="orientation">
                                        <enum>Qt::Horizontal</enum>
                                        </property>
                                        
                                        <property name="sizeType">
                                            <enum>QSizePolicy::Fixed</enum>
                                        </property>
                                        
                                        <property  name="sizeHint"  stdset="0">
                                        <size>
                                            <width>30</width>
                                            <height>20</height>
                                        </size>
                                        </property>
                                    </spacer>
                                    </item>
                                    <item>
                                    <widget  class="QCheckBox"  name="check_all">
                                        <property  name="text">
                                        <string>All</string>
                                        </property>
                                    </widget>
                                    </item>
                                    <item>
                                    <widget  class="QCheckBox"  name="check_inv">
                                        <property  name="text">
                                        <string>Invert</string>
                                        </property>
                                    </widget>
                                    </item>
                                </layout>
                            </widget>
                            </item>
                            
                            <item>  
                                <layout  class="QHBoxLayout"  name="hlSelColButton">

                                    <item>
                                        <spacer  name="horizontalSpacer_8">
                                        <property  name="orientation">
                                            <enum>Qt::Horizontal</enum>
                                        </property>
                                        
                                        <property  name="sizeHint"  stdset="0">
                                            <size>
                                            <width>30</width>
                                            <height>20</height>
                                            </size>
                                        </property>
                                        </spacer>
                                    </item>
                                    
                                    <item>
                                        <widget  class="QPushButton"  name="selColBtn">
                                        <property  name="text">
                                            <string>Select</string>
                                        </property>
                                        </widget>
                                    </item>
                                
                                </layout>
                            </item>
                            
                        </layout>
                        </widget>
                    </item>
                    

                    <item>
                        <widget  class="QFrame"  name="wFilterData">
                        <property  name="frameShape">
                            <enum>QFrame::StyledPanel</enum>
                        </property>
                        <property  name="frameShadow">
                            <enum>QFrame::Raised</enum>
                        </property>
                        <property  name="lineWidth">
                            <number>8</number>
                        </property>

                        <layout  class="QVBoxLayout"  name="vlFilterData">
                            
                            <item>
                            <widget  class="QLabel"  name="label_select">
                                <property  name="text">
                                <string>Filter Data</string>
                                </property>
                            </widget>
                            </item>
                            
                            <item>
                            <layout  class="QHBoxLayout"  name="horizontalLayout_5">
                                <item>
                                <spacer  name="horizontalSpacer_5">
                                    <property  name="orientation">
                                    <enum>Qt::Horizontal</enum>
                                    </property>
                                    
                                    <property name="sizeType">
                                        <enum>QSizePolicy::Fixed</enum>
                                    </property>
                                    
                                    <property  name="sizeHint"  stdset="0">
                                    <size>
                                        <width>30</width>
                                        <height>20</height>
                                    </size>
                                    </property>
                                </spacer>
                                </item>
                                <item>
                                <layout  class="QVBoxLayout"  name="vlComboFilter"/>
                                </item>
                            </layout>
                            </item>
                            
                            <item>
                                <widget  class="QWidget"  name="wFilterCategorical">
                                    <layout  class="QHBoxLayout"  name="hlFilterCategorical">
                                        <item>
                                            <spacer  name="horizontalSpacer">
                                                <property  name="orientation">
                                                <enum>Qt::Horizontal</enum>
                                                </property>
                                                
                                                <property name="sizeType">
                                                    <enum>QSizePolicy::Fixed</enum>
                                                </property>
                                                
                                                <property  name="sizeHint"  stdset="0">
                                                <size>
                                                    <width>30</width>
                                                    <height>20</height>
                                                </size>
                                                </property>
                                            </spacer>
                                        </item>

                                        <item>
                                            <layout  class="QHBoxLayout"  name="hlFilterCat">
                                            </layout>
                                        </item>
                                        
                                        
                                    </layout>
                                </widget>
                            </item>
                                

                            <item>
                            <widget  class="QWidget"  name="wFilterNumerical">
                                
                                <layout  class="QVBoxLayout"  name="hlFilterNumeric">

                                <item>
                                    <layout  class="QHBoxLayout"  name="hlFilterNumeric">
                                        <item>
                                            <spacer  name="horizontalSpacer">
                                                <property  name="orientation">
                                                <enum>Qt::Horizontal</enum>
                                                </property>
                                                
                                                <property name="sizeType">
                                                    <enum>QSizePolicy::Fixed</enum>
                                                </property>
                                                
                                                <property  name="sizeHint"  stdset="0">
                                                <size>
                                                    <width>30</width>
                                                    <height>20</height>
                                                </size>
                                                </property>
                                            </spacer>
                                        </item>
                                        
                                        <item>
                                            <widget  class="QLabel"  name="lMinVal">
                                                <property  name="text">
                                                    <string>Min: </string>
                                                </property>
                                            </widget>
                                        </item>

                                        <item>
                                            <widget  class="QLineEdit"  name="edit_minval">
                                            </widget>
                                        </item>
                                    </layout>
                                </item>
                                
                                <item>
                                    <layout  class="QHBoxLayout"  name="hlFilterNumeric">
                                        <item>
                                            <spacer  name="horizontalSpacer">
                                                <property  name="orientation">
                                                <enum>Qt::Horizontal</enum>
                                                </property>
                                                
                                                <property name="sizeType">
                                                    <enum>QSizePolicy::Fixed</enum>
                                                </property>
                                                
                                                <property  name="sizeHint"  stdset="0">
                                                <size>
                                                    <width>30</width>
                                                    <height>20</height>
                                                </size>
                                                </property>
                                            </spacer>
                                        </item>
                                    
                                        <item>
                                            <widget  class="QLabel"  name="lMaxVal">
                                                <property  name="text">
                                                    <string>Max: </string>
                                                </property>                                
                                            </widget>
                                        </item>

                                        <item>
                                            <widget  class="QLineEdit"  name="edit_maxval">
                                            </widget>
                                        </item>                        

                                    </layout>
                                </item>
                                </layout>
                                    

                            </widget>
                            </item>
                            
                            <item>  
                                <layout  class="QHBoxLayout"  name="hlFilterButton">

                                    <item>
                                        <spacer  name="horizontalSpacer_8">
                                        <property  name="orientation">
                                            <enum>Qt::Horizontal</enum>
                                        </property>
                                        
                                        <property  name="sizeHint"  stdset="0">
                                            <size>
                                            <width>40</width>
                                            <height>20</height>
                                            </size>
                                        </property>
                                        </spacer>
                                    </item>
                                    
                                    <item>
                                        <widget  class="QPushButton"  name="filterBtn">
                                        <property  name="text">
                                            <string>Filter</string>
                                        </property>
                                        </widget>
                                    </item>
                                
                                </layout>
                            </item>
                            
                            
                        </layout>
                        </widget>
                    </item>
                    
                    <item>
                        <widget  class="QFrame"  name="wDropDupl">
                        <property  name="frameShape">
                            <enum>QFrame::StyledPanel</enum>
                        </property>
                        <property  name="frameShadow">
                            <enum>QFrame::Raised</enum>
                        </property>
                        <property  name="lineWidth">
                            <number>8</number>
                        </property>

                        <layout  class="QVBoxLayout"  name="vlDropDupl">
                            
                            <item>
                            <widget  class="QLabel"  name="label_select">
                                <property  name="text">
                                <string>Drop Duplicates</string>
                                </property>
                            </widget>
                            </item>
                            
                            <item>
                            <layout  class="QHBoxLayout"  name="horizontalLayout_5">
                                <item>
                                <spacer  name="horizontalSpacer_5">
                                    <property  name="orientation">
                                    <enum>Qt::Horizontal</enum>
                                    </property>

                                    <property name="sizeType">
                                        <enum>QSizePolicy::Fixed</enum>
                                    </property>

                                    <property  name="sizeHint"  stdset="0">
                                    <size>
                                        <width>30</width>
                                        <height>20</height>
                                    </size>
                                    </property>
                                </spacer>
                                </item>
                                <item>
                                <layout  class="QVBoxLayout"  name="vlComboSelDupl"/>
                                </item>
                            </layout>
                            </item>

                            <item>  
                                <layout  class="QHBoxLayout"  name="hlFilterButton">

                                    <item>
                                        <spacer  name="horizontalSpacer_8">
                                        <property  name="orientation">
                                            <enum>Qt::Horizontal</enum>
                                        </property>
                                        <property  name="sizeHint"  stdset="0">
                                            <size>
                                            <width>40</width>
                                            <height>20</height>
                                            </size>
                                        </property>
                                        </spacer>
                                    </item>
                                    
                                    <item>
                                        <widget  class="QPushButton"  name="dropBtn">
                                        <property  name="text">
                                            <string>Drop</string>
                                        </property>
                                        </widget>
                                    </item>
                                
                                </layout>
                            </item>

                        </layout>
                        </widget>
                    </item>
                            
                    <item>
                        <spacer  name="verticalSpacer">
                        <property  name="orientation">
                            <enum>Qt::Vertical</enum>
                        </property>
                        <property  name="sizeHint"  stdset="0">
                            <size>
                            <width>20</width>
                            <height>40</height>
                            </size>
                        </property>
                        </spacer>
                    </item>
                                    

                    </layout>
                </widget>
                </widget>
            </item>
            </layout>
        
        </widget>
      </item>

    </layout>
        
  </widget>
  <resources/>
  <connections/>
</ui>
